components/OrganizationEdit/OrganizationEdit.js

```diff
@@ -184,11 +186,22 @@ class OrganizationEdit extends Component {
     this.setState(update)
   }
 
-  handleLocations = (newLocation, index) => {
+  handleLocations = (newLocation, index, save) => {
     const { locations } = this.state
     const newLocations = [...locations]
 
     newLocations[index] = newLocation
+    
+    if (save) {
+      updateLocation(newLocation).then(response => {
+        let success = !response.hasOwnProperty('error')
+        this.setState({
+          hasSubmit: true,
```

Comment:
is this used anywhere? I don't see it anywhere else in the diff

Reply:
`hasSubmit` is used to put up a the success/failure banner.